modules/acid/src/main/ts/ephox/acid/api/colour/RgbaColour.ts

```diff
@@ -102,13 +105,20 @@ const fromStringValues = (red: string, green: string, blue: string, alpha: strin
   const g = parseInt(green, 10);
   const b = parseInt(blue, 10);
   const a = parseFloat(alpha);
+
   return rgbaColour(r, g, b, a);
 };
 
-const fromString = (rgbaString: string): Optional<Rgba> => {
-  if (rgbaString === 'transparent') {
-    return Optional.some(rgbaColour(0, 0, 0, 0));
+const getColorFormat = (colorString: string): string => {
```

Comment:
This return should've been a sum type, not a `string`.

All I really wanted was some sort of function to run `rgbRegex.test` - not a full "get format" method when we only ever want to check one of the conditions...